src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -1377,6 +1381,7 @@ impl EventProcessor {
                     event: WindowEvent::CursorMoved {
                         device_id: mkdid(util::VIRTUAL_CORE_POINTER),
                         position: location.cast(),
+                        r#type: CursorType::Mouse,
                     },
                 };
                 callback(&self.target, event);
```

Comment:
> Mouse cursor position changes when touch events are received.
> Only the first concurrently active touch ID moves the mouse cursor.

This is a bit confusing to me, especially because I believe it doesn't map to other backends. Why do we tell the user that the mouse moves if the cursor moves with touch input?
Mouse input is cursor movement, but not all cursor movement is mouse input.

I'm really not sure what the purpose of this is.